### DISCUSSION TITLE
Gaia Testnet (v12.0.0-rc0)ICA Delegation Ack Error

### DISCUSSION THREAD

**Discussion by cloudgray** [cosmos/community#47] [Chain Builders (Cosmos SDK)]
Hello, guys.

When making an ICA Delegate request on the gaia testnet, an Acknowledgement Error is occurring, and it comes with an error code of 107. This error code 107 doesn't seem to be returned from the ibc-go, cosmos-sdk, or cometbft packages. I suspect it might be arising during the marshalling process of the ack in protobuf, but pinpointing the exact cause is challenging.

Interestingly, when setting up the gaia 12.0.0-rc0 version locally and launching the network, everything works without any issues. Other features like ICQ and IBC Transfer are also functioning without problems.

Our team suspects that the issue might be related to permission problems caused by the lsm feature added in the 12.0.0-rc0 version. We speculate that only consumer chains that have registered their pubkey and denom via the gaia testnet's cli can delegate.

We are seeking insights from appchain experts.

P.S.
Our primary dependencies are as follows:

github.com/cosmos/cosmos-sdk v0.47.3
github.com/cometbft/cometbft v0.37.2
github.com/cosmos/ibc-go/v7 v7.2.0
github.com/cosmos/gogoproto v1.4.10

<img width="1024" alt="스크린샷 2023-09-12 오전 10 34 16" src="https://github.com/cosmos/community/assets/39086446/f6cb9a55-11fb-4c6c-b0f2-ef1310cd75d5">
<img width="1024" alt="스크린샷 2023-09-11 오후 2 24 23" src="https://github.com/cosmos/community/assets/39086446/acd1a00a-9a7c-4c63-beba-038b5052ba35">

**Comment by MSalopek** [comment]
Gaia has moved on to [v13](https://github.com/cosmos/gaia/releases/tag/v13.0.0).

The testnet has been running v13 for a while now.

Please consider executing your tests with the latest version and let us know if the problem persists.

> Our team suspects that the issue might be related to permission problems caused by the lsm feature added in the 12.0.0-rc0 version. We speculate that only consumer chains that have registered their pubkey and denom via the gaia testnet's cli can delegate.

I'm not sure that can be the case. Consumer key assignment only affects consensus keys that are used on the network and reward denoms only affect reward packets sent from the consumer chains.
